.circleci/config.yml

```diff
@@ -772,6 +784,7 @@ jobs:
     executor: bionic
     environment:
       EMTEST_SKIP_NODE_CANARY: "1"
+      EMTEST_SKIP_WASM64: "1"
```

Comment:
is it necessary to have both this and the node canary update?

Reply:
This builder does not have node canary installed.   So it can no longer run wasm64 tests after this change.   We could install node canary, but general we only configure one version of node per bot.